README.txt

```diff
@@ -0,0 +1,72 @@
+This branch is the github pages repo for armadaproject.io. This document
```

Comment:
Any reason why this can't be a markdown file?

Reply:
Didn't want it to be published at armadaproject.io/readme, and this was the most obvious route to go to prevent that.